requirements/edx/django.txt

```diff
@@ -1 +1,2 @@
 django==2.2.24
+django-cookies-samesite==0.9.0
```

Comment:
https://github.com/edx/configuration/blob/d85d945a3c0a3ba505fe342a714f194d8919b15d/playbooks/roles/edxapp/defaults/main.yml#L1708

It will deploy from here.